forms-flow-web/src/components/PrivateRoute.jsx

```diff
@@ -298,18 +299,21 @@ const PrivateRoute = React.memo((props) => {
                 component={ServiceFlow}
               />
             )}
-
+            <Route exact path={`${redirecUrl}admin`} /> 
```

Comment:
why we need to add this here ?

Reply:
Without this route , when navigates '/admin' the path may be not taken as valid, leads to 404 page.